### DISCUSSION TITLE
Ranking OD adjust in the difficulty adjust mod

### DISCUSSION THREAD

**Discussion by kala12111** [ppy/osu#25765] [Ideas] [answered]
I can see why some issues could be caused with some of the adjustments being added to ranked. I feel like OD adjust is pretty harmless and perhaps only ranking it if you raise the OD and not lowering it would be nice as i can see cheesing being an issue (raketapping etc.). As someone who enjoys EZ mostly this would be nice as i enjoy the low ar, but the OD means it's very hard to gain rank if you play this mod.

I'm sorry, if this has been mentioned before somewhere and i didn't see it. But it'd be nice to know other opinions on this

**Comment by Flamiii** [comment, marked as answer]
EZ and HR are incompatible with Difficulty Adjust because they directly affect map stats and they would contradict your Difficulty Adjust settings, so even if OD adjust was ranked, you would not be able to use it in the way that you want to. If you wanted the low CS and AR of EZ but with a high OD, Difficulty Adjust as a whole would have to be ranked.

**Reply by kala12111** [reply]
Yeah i think you're right. I disagree with Difficulty adjust as a whole being ranked and i guess this is too niche of a idea to warrant separating OD adjust from difficulty adjust as a whole to allow for use with mods.
